ansible_collections/arista/avd/plugins/plugin_utils/strip_empties.py

```diff
@@ -41,7 +42,15 @@ def strip_null_from_data(data, strip_values_tuple=(None,)):
     return data
 
 
-def strip_empties_from_list(data, strip_values_tuple=(None, "", [], {},)):
+def strip_empties_from_list(
+    data,
+    strip_values_tuple=(
```

Comment:
Maybe the default value can be a global var in this file given that it is used for two functions (though probably not as part of this PR)

Reply:
Not in this PR. It is busy enough as it is :)